src/components/Documentation/DocLayout/SidebarMenu/index.js

```diff
@@ -0,0 +1,131 @@
+import React, { useEffect, useRef, useState } from 'react'
```

Comment:
you don't need `Doc` prefix in this case?

Reply:
Fixed